litmus-portal/backend/graphql-server/pkg/file_handlers/file_handlers.go

```diff
@@ -0,0 +1,84 @@
+package file_handlers
+
+import (
+	"bufio"
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/utils"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/pkg/cluster"
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/pkg/database/mongodb"
+)
+
+//ManifestParser parses manifests yaml and generates dynamic manifest with specified keys
+func manifestParser(id, key, server, template string) ([]string, error) {
```

Comment:
I move the manifestParse() from pkg/cluster because having one function in a file doesn't help.
This is the right place where we can use the manifestParser()